### PR TITLE
Add persistent CLI history

### PR DESCRIPTION
**Pull request by eolivelli**
- add CLI history, stored on user HOME directory
- add flag -nsch to disable the history
- fix a problem with password that were shown in the logs or on the CLI using '-v' option
- add 'setschema' command to change default schema
- show current schema on the prompt

### CODE REVIEW

File: herddb-cli/src/main/java/herddb/cli/HerdDBCLI.java [hunk 1]
```diff
@@ -1595,23 +1607,32 @@ private static void println(Object msg) {
...
-        LineReader reader = LineReaderBuilder.builder()
+        LineReaderBuilder readerBuilder = LineReaderBuilder.builder()
                 .history(new DefaultHistory())
-                .terminal(terminal)
-                .build();
-        String prompt = "herd: ";
+                .terminal(terminal);
+        if (persistHistory) {
+            String historyDirectory = System.getProperty("user.home", ".");
+            File historyFile = new File(historyDirectory, ".herddb.cli.history");
```
Comment (amitvc) [inline herddb-cli/src/main/java/herddb/cli/HerdDBCLI.java]: How long can this file grow?

Comment (eolivelli) [reply herddb-cli/src/main/java/herddb/cli/HerdDBCLI.java]: It is unlimited 
We could set a bound 

https://www.javadoc.io/doc/org.jline/jline/3.5.1/org/jline/reader/LineReader.html#HISTORY_FILE_SIZE

Comment (eolivelli) [reply herddb-cli/src/main/java/herddb/cli/HerdDBCLI.java]: I have added 100 as bound.

**Comment by eolivelli** [comment]
@amitvc  I have addressed your comment.
Please take a look and merge if you are okay

**Comment by eolivelli** [comment]
@amitvc I have merged this branch.
Please reopen if you are not okay

we are close to the release

**Pull request merged**